docroot/themes/custom/twentyeighteen/sass/base/_form.scss

```diff
@@ -9,6 +9,10 @@ form {
   margin: 0 0 1em;
 }
 
+form .form-item {
+  padding: 0.3em;
```

Comment:
I think it needs a little more padding. Maybe 0.5em or 0.75em (I don't want it to look too clobbered).